src/index.js

```diff
@@ -5,19 +5,37 @@ import bannerPlugin from '@comandeer/babel-plugin-banner';
 import { getCommentContent } from '@comandeer/babel-plugin-banner/utils';
 import { transform } from 'babel-core';
 
+// NOTE: lodash or something else more precise can be an option here
+const isString = ( v ) => {
+	return v != null && typeof v === 'string';
+};
+const isFn = ( v ) => {
+	return v != null && typeof v === 'function';
+};
+const isFnOrString = ( v ) => {
+	return isString( v ) || isFn( v );
+};
+
 function babili( options = {} ) {
+	let _banner;
```

Comment:
Well, it's the only thing I don't like: this variable name :D It's easily confusable with later `banner` one and does not really explain the purpose of this variable. Maybe something like `rollupBanner` to indicate that it comes from rollup options?